python/execute.py

```diff
@@ -306,27 +312,39 @@ def visit_target(self, query):
     def visit_join(self, query):
         raise "Not implemented"
 
+    @staticmethod
+    def _visit_driver_obj(it, key, driver):
+        with ExitStack() as stack:
+            for i in it:
+                id_i = i[":id"]
```

Comment:
It was a bit hard to move it to driver since it's used in places like item.py and walk.py.
I made it a named constant (item.IDKEY) instead, so it becomes easy to change if necessary.